### PR TITLE
Add: Question page with functionality

### PR DESCRIPTION
**Pull request by saurabhchirde**
Feature added

1. Question Page
2. Auto Timer of 20sec, after which the next question will be shown.
3. Right and Wrong answer Highlight after selecting an option.
4. Rules Modal
5. Final Result Modal

question

### CODE REVIEW

**Review by PuneetP16 — APPROVED** [review]
UI Suggestion: Background could be more opaque better readability.

image

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
   text-align: center;
+  background-color: var(--background-color);
+}
+
+.modal-backdrop {
+  background-color: rgba(0, 0, 0, 0.447);
+  width: 100%;
+  height: 120vh;
+  position: fixed;
+  z-index: 3;
```
Comment (PuneetP16) [inline src/App.css:11]: Suggestion: You could use variables for z-index for making it more manageable at one place.

File: src/Components/Cards/QuestionCard/QuestionCard.css [hunk 2]
```diff
@@ -0,0 +1,222 @@
+.question-card,
+.question,
+.question-details,
+.quiz-detail,
+.quiz-total-time,
+.quiz-high-score,
+.quiz-progress {
+  display: flex;
+}
+
```
Comment (PuneetP16) [inline src/Components/Cards/QuestionCard/QuestionCard.css:10]: Suggestion: You could directly provide flex property to the html tags using flex utility class.

File: src/Pages/Question/QuestionPage.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+.question-section > div {
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+@media screen and (max-width: 1024px) {
+  .question-page-content {
+    padding: 2rem 1.5rem 1rem;
+  }
+
```
Comment (PuneetP16) [inline src/Pages/Question/QuestionPage.css:60]: Suggestion: keep consistent spacing between declaring two css properties.

**Review by PuneetP16 — APPROVED** [review]
Good work on UI, check the comments. Good to merge.

**Review by m26unkwn — APPROVED** [review]
UI is clean and and beautiful
Love the minimalistic design

1. code is clean and readable
2. Naming is good
3. you can use sematic html
4. css is well written

**Pull request merged**